app/assets/stylesheets/components/_super-navigation-header.scss

```
@@ -0,0 +1,90 @@
+$breakpoint: 769px;
+
+// Remove bottom borders
+.govuk-header__super-navigation,
```

The `govuk` prefix is the namespace for the Design System - anything in Static should be using the prefix `app-`.
```suggestion
.app-header__super-navigation,
```

---

Updated